enterprise/signals.py

```diff
@@ -315,6 +332,15 @@ def delete_enterprise_catalog_data(sender, instance, **kwargs):     # pylint: di
             exc_info=exc
         )
 
+    customer = instance.enterprise_customer
+    for channel in INTEGRATED_CHANNELS:
+        if channel.objects.filter(enterprise_customer=customer, active=True).exists():
+            try:
+                mark_orphaned_content_metadata_audit.delay()
+                break
+            except Exception as exc:  # pylint: disable=broad-except
+                logger.exception(f'Failed to mark orphaned content metadata audits. Task failed with exception: {exc}')
```

Comment:
nit: you won't actually know if the task failed in this context - that task is running asynchronously off in a worker process somewhere else.  The only kind of exception we'd catch here is about a problem submitting the task - maybe update the log message to reflect that?  But just straight invoking the task without this `try/except` would be fine, too.  We probably want this to fail loudly since it's not user-facing and is a very sparsely occurring event.

Reply:
I'm fine just taking the try/except out